refl1d/probe.py

```diff
@@ -1368,7 +1372,7 @@ def load4(filename, keysep=":", sep=None, comment="#", name=None,
         if any(isinstance(d, QProbe) for d in xs if d is not None):
             probe = PolarizedQProbe(xs, Aguide=Aguide, H=H)
         else:
-            probe = PolarizedNeutronProbe(xs, Aguide=Aguide, H=H)
+            probe = PolarizedNeutronProbe(xs, Aguide=Aguide, H=H, oversampling=oversampling)
```

Comment:
In the current form, it does accept a seed which is stored as an instance attribute - I was wondering, though: why does e.g. QProbe.oversample have a default seed value of 1 instead of None?  Woudn't it be preferable to shake the dice if the user doesn't specify a seed value explicitly?